src/Mvc/src/Microsoft.AspNetCore.Mvc.Razor/Microsoft.AspNetCore.Mvc.Razor.csproj

```diff
@@ -20,8 +20,8 @@
     <Reference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" />
 
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" Version="$(MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion)" />
```

Comment:
Should this change to `<Reference>` too?

Reply:
If it's just props, we can `<Import>` it. 

I think this is the source of the NU1605 error.

Reply:
We would have to do that every place the package get used (transitively) which is fairly substantial. Rolling back the version to 2.1 seems to get the build to work correctly. Is there a reason we wouldn't do that?

Reply:
As long as dropping MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion back to 2.1.1 doesn't regress the product in anyway, that's fine with me.

Would putting that import in the repo root Directory.Build.props be safe?

Reply:
We spoke about this offline. Referencing the Razor targets in the build is problematic since it would cause the tasks assemblies to get loaded by msbuild running the tests. We aren't going to make breaking changes to these and the Razor.Design tests should be sufficient to verify the sanity of these targets.